src/components/criterion/FilterGreaterThan.vue

```diff
@@ -26,7 +26,7 @@ export default Vue.component("filter-greater-control", {
         pillFormatter: {
             type: Function,
             default: filterDefinition =>
-                `${filterDefinition.field} >= ${filterDefinition.threshold}`
+                `${filterDefinition.field} ≥ ${filterDefinition.threshold}`
```

Comment:
just curious, but does this actually work? I think ≥ is only an html entity and not an actual check.

Reply:
you're right, didn't see the **` `**, which means it just a string formatter, so it's indeed just a html entity.